gst/tensor_decoder/tensordec.c

```diff
@@ -761,11 +762,19 @@ gst_tensordec_label_set_output (GstTensorDec * self, GstBuffer * outbuf,
   g_assert (out_mem != NULL);
   g_assert (gst_memory_map (out_mem, &out_info, GST_MAP_WRITE));
 
+  /**
+   * Security: strncpy protects from buffer overflows. But, if it prevents an
+   * overflow without null termiating, a subsequent string operation cause overflow.
+   *
+   * Gstreamer: Note that gst-plugins-base/gstbasetextoverlay.c:g_utf8_validate()
```

Comment:
To Reviewes, 
We are going to follow up the design structure of gstreamer instead we try to use `memset(destination, 0, len)` libary call to pass a security inspection (e.g., not null termination issue in Svace).